lua/bufferline.lua

```diff
@@ -233,7 +235,10 @@ function bufferline.setup(options)
 
   create_user_command(
     'BufferClose',
-    function(tbl) bbye.bdelete(tbl.bang, tbl.args, tbl.mods) end,
+    function(tbl)
+      local focus_buffer = state.find_next_buffer(get_current_buf())
```

Comment:
Re: https://github.com/romgrk/barbar.nvim/pull/270#discussion_r933993085

This would be `require'bufferline.state'.find_next_buffer(…)`